### PR TITLE
Build hierarchy in frontend

### PR DESCRIPTION
**Pull request by inglesp**
This is to allow more information to be shown in tooltips

### CODE REVIEW

**Comment by inglesp** [comment]
@ghickman this might affect your work on the tree view -- if it's going to complicate things for you, let me know.

File: codelists/tests/test_presenters.py [hunk 1]
```diff
@@ -124,59 +123,53 @@ def test_tree_tables():
     #   └  239964003 (Soft tissue lesion of elbow region)
 
     assert presenters.tree_tables(
-        ancestor_codes, hierarchy, code_to_term, code_to_type, code_to_status
+        ancestor_codes, hierarchy, code_to_term, code_to_type
     ) == [
         {
             "heading": "Disorder",
+            # fmt: off
```
Comment (ghickman) [inline codelists/tests/test_presenters.py:130]: Is black collapsing the dictionaries into single lines?

**Review by ghickman — APPROVED** [review]
👍

I think this might even simplify my tree work!

File: codelists/tests/test_presenters.py (hunk 1 above)
Comment (inglesp) [reply codelists/tests/test_presenters.py:130]: It's doing it for some lines and not for others

```
            "rows": [
                {"code": "128133004", "term": "Disorder of elbow", "pipes": [],},
                {"code": "429554009", "term": "Arthropathy of elbow", "pipes": ["├"],},
                {
                    "code": "439656005",
                    "term": "Arthritis of elbow",
                    "pipes": ["│", "└"],
                },
                {
                    "code": "202855006",
                    "term": "Lateral epicondylitis",
                    "pipes": ["│", " ", "└"],
                },
                {
                    "code": "35185008",
                    "term": "Enthesopathy of elbow region",
                    "pipes": ["├"],
                },
                {"code": "73583000", "term": "Epicondylitis", "pipes": ["│", "└"],},
                {
                    "code": "202855006",
                    "term": "Lateral epicondylitis",
                    "pipes": ["│", " ", "└"],
                },
                {
                    "code": "239964003",
                    "term": "Soft tissue lesion of elbow region",
                    "pipes": ["└"],
                },
            ],
```

Comment (ghickman) [reply codelists/tests/test_presenters.py:130]: The second to last comma on this line:

```{"code": "128133004", "term": "Disorder of elbow", "pipes": [],},```

_should_ make Black split up the items per line (or rather, leave it how you wrote it!), do you have black==20.8b1 installed?  It was supposed to fix this kind of error!

**Pull request merged**